ci/actions-templates/linux-builds-template.yaml

```diff
@@ -152,6 +163,15 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         if: github.event_name == 'push' && github.ref == 'refs/heads/stable'
+      - name: Build a snapcraft configuration file
+        run: |
+          export DO_SNAP=1
+          export SNAP_ARCH=${{ matrix.snap_arch }}
+          export SNAP_EDGE=1 # skip-stable
+          sh ci/snapcraft.sh
+        if: matrix.snap_arch != ''
```

Comment:
```suggestion
        if: matrix.snap_arch
```
here and there

Reply:
I prefer the explicit comparison since it's meant to be a string if it's set.